layouts/partials/home/hero-image-rendition.html

```diff
@@ -0,0 +1,20 @@
+{{- $heroData := slice -}}
+{{- range .Params.images -}}
+  {{- $image := resources.Get (strings.TrimPrefix "/" .) -}}
```

Comment:
Wait if the `1.webp` was not moved, does this work? I don't know what `resources.Get` searches

Related - why were the webp files moved, and not all of them. I presume "cos it didn't work otherwise"?
Oh and you didn't move them, as I didn't see the other files deleted? Can you clarify?

Reply:
I had meant to put _all_ the webp files into `assets/images/hero`. 1.webp was already there wasn't it?